chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1937,3 +1941,88 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> List[CoinRecord]:
             assert result[1] != MempoolInclusionStatus.FAILED
     except ValidationError as e:
         assert e.code == expected
+
+
+@pytest.mark.anyio
+@pytest.mark.limit_consensus_modes(allowed=[ConsensusMode.HARD_FORK_2_0])
+@pytest.mark.parametrize(
+    "item_cost, max_block_clvm_cost, expected_block_items, expected_block_cost",
+    [(144_785_330, 144_785_330 * 2, 2, 289_508_831), (144_785_330, 144_785_330 * 2 - 1, 1, 144_785_330)],
+)
```

Comment:
`item_cost` is a constant. making it a parameter here just obfuscates the important parameters to this test.
I think you should give it a name and just make it a constant.

Reading this test, it's not obvious why it's correct (or really, what it's testing). I think you need more comments describing it. For example, what the two cases you run tests.

I also think it would be helpful to explain why the case that includes two transactions has a cost that's less than 2x the single transaction. (I imagine it's compression).

If you give the item cost constant a name, you could write the expected block cost as `ITEM_COST * 2 - 61829` instead, to highlight the diff from the intuitively expected value.

If you intend to extend this test with additional cases where item_cost is different, I still think it would make sense to give those names, since the parameters are near-multiples of the item cost